### PR TITLE
targets for logging stack

### PR DESCRIPTION
**Pull request by pb82**
make targets for the logging stack based on the resoruces from https://github.com/redmikhail/kafka-logging

### CODE REVIEW

File: install/Makefile [hunk 1]
```diff
@@ -12,6 +12,9 @@ export GLOBAL_THANOS_RECEIVER ?= kafka-thanos-receiver
 export GLOBAL_THANOS_QUERIER ?= kafka-thanos-querier
 export GLOBAL_GRAFANA ?= kafka-grafana-global
 
+export LOGS_MAX_AGE ?= 1d
+export LOGS_STORAGE ?= 60G
```
Comment (david-martin) [inline install/Makefile:16]: Wondering if this is OK for OSD stage where there's 100G quota.
Some calculations:

* 5Gi x 3 (kafka brokers) = 15Gi
* 5Gi x 3 (zookeepers) = 15Gi
* 60Gi x 1? = 60Gi

Thats 90Gi for just monitoring/logging pieces (provided it's 1 pod/replica for the 60Gi)

@davidffrench @damienomurchu Is there anything else that needs a PV at this time?
I see the values in here as a short term solution, for dev clusters only

Comment (Rajagopalan-Ranganathan) [reply install/Makefile:16]: I guess there is Prometheus as well. (but not for the demo may be).

File: install/resources/logging/Makefile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+.PHONY: create/operatorgroups
+create/operatorgroups:
+	@echo "create logging operator groups"
+	@cat ./operatorgroups.yaml | oc apply -f -
+
+.PHONY: create/subscriptions
+create/subscriptions:
+	@echo "create logging operator subscriptions"
+	@cat ./subscriptions.yaml | oc apply -f -
+
```
Comment (Rajagopalan-Ranganathan) [inline install/resources/logging/Makefile:15]:
```error: unable to recognize "STDIN": no matches for kind "ClusterLogging" in version "logging.openshift.io/v1"
make[1]: *** [Makefile:19: create/logging] Error 1
make[1]: Leaving directory '/home/rarangan/pb82/kafka-monitoring-stuff/install/resources/logging'
make: *** [Makefile:84: install/logging] Error 2``` - need some waits. works fine on the second run.

**Comment by Rajagopalan-Ranganathan** [comment]
Verified in an OSD cluster with the manual steps in section 9 of: https://docs.google.com/document/d/1oPadU1DMdiYr8F0DgxMe5jSmPvp1BGw9xAJw_XyRazM/edit# and commands to taint and label a worker node: ```oc adm taint node <node-name> openshift-logging/storage=true:NoSchedule``` ```oc label node <node-name> openshift-logging/storage=true```

**Comment by Rajagopalan-Ranganathan** [comment]
image

**Comment by Rajagopalan-Ranganathan** [comment]
image

File: install/Makefile (hunk 1 above)
Comment (davidffrench) [reply install/Makefile:16]: So sorry @david-martin , only seeing this now. There is no storage limit for CCS clusters.

**Review by david-martin — APPROVED** [review]

**Comment by Rajagopalan-Ranganathan** [comment]
with manually killing a couple of zookeeper pods, we can get a few errors and warning in kibana dashboard: 
image

**Pull request merged**